string orderId
string city
float32 i
float32 j